### ISSUE TITLE
/nix folder in root on Mac OS would be more friendly if marked as hidden

### ISSUE THREAD

**Issue by copumpkin** [NixOS/nix#483]
This boils down to calling `chflags hidden /nix` during the install script. This sets an extended attribute and hides it in Finder, without changing the path. Other unixy paths of that sort living in the root get hidden in a similar way in standard Mac OS.

**Comment by copumpkin** [comment]
Might need to wait for us to build our own `chflags` to improve purity :smile:

**Comment by edolstra** [comment]
Sounds good. The installer script just uses whatever is in $PATH btw, so we can just use `chflags` if we want.

**Comment by copumpkin** [comment]
Yeah, it could probably do that for now, but an ideal installer would bundle everything it used (it already does that on Linux doesn't it?)

**Comment by edolstra** [comment]
No it doesn't. After all, you can't use the bundled stuff until you've unpacked it, so you need external utilities anyway.

**Comment by copumpkin** [comment]
Oh, I figured it would be like the statically linked busybox-flavored things in the bootstrap tools, and we'd only rely on a host curl/shell. Probably overkill since we still need those :smile:

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
